politeiad/plugins/pi/pi.go

```diff
@@ -168,8 +171,27 @@ const (
 	// is not one of the supported domains.
 	ErrorCodeProposalDomainInvalid ErrorCodeT = 11
 
+	// ErrorCodeTokenInvalid is returned when a record token is
+	// provided as part of a plugin command payload and is not a valid
+	// token or the payload token does not match the token that was
+	// used in the API request.
+	ErrorCodeTokenInvalid ErrorCodeT = 12
+
+	// ErrorCodePublicKeyInvalid is returned when a public key is not
+	// a valid hex encoded, Ed25519 public key.
+	ErrorCodePublicKeyInvalid ErrorCodeT = 13
+
+	// ErrorCodeSignatureInvalid is returned when a signature is not
+	// a valid hex encoded, Ed25519 signature or when the signature is
+	// wrong.
+	ErrorCodeSignatureInvalid ErrorCodeT = 14
+
+	// ErrorCodeBillingStatusAlreadySet is returned when a billing status
+	// was already set.
+	ErrorCodeBillingStatusAlreadySet = 15
```

Comment:
I would change this to `BillingStatusChangeNotAllowed` then return the specific reason that its not allowed (i.e. its already been set) in the `ErrorContext`.

The reason I would do it this way is because not allowing more than 1 billing status change is an implementation detail that could change in the future.  If it does change, this error code doesn't make a lot of sense.  `BillingStatusChangeNotAllowed` is also more generic and can be re-used in future for other implemenation details.

Reply:
makes sense 👍🏽